### DISCUSSION TITLE
[Unstable] Lighting is broken in leveleditor

### DISCUSSION THREAD

**Discussion by ubertpendragon** [FakeFishGames/Barotrauma#14741] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

The shape of the submarine's shadow appears in level editor if you enable lighting

![image](https://github.com/user-attachments/assets/71c10faa-1c75-4b46-b24e-91772cf41983)

### Reproduction steps

1. leveleditor
2. enable lighting

### Bug prevalence

Just once

### Single player or multiplayer?

Single player

### Version

v1.6.4.0 (unstable)

### Which operating system did you encounter this bug on?

Windows

**Comment by itchyOwl** [comment]
Cannot repro.

**Comment by ubertpendragon** [comment]
Ok I it happened disabling lighting with console command + enable lighting in level editor, not that big of a deal
